tests/pkg_mbedtls/Makefile

```diff
@@ -0,0 +1,55 @@
+include ../Makefile.tests_common
+
+# The following boards currently miss entropy sources.
+BOARD_BLACKLIST :=  6lowpan-clicker \
+                    atxmega-a1u-xpro \
+                    atxmega-a3bu-xplained \
+                    avsextrem \
+                    cc1312-launchpad \
+                    cc1350-launchpad \
+                    cc1352-launchpad \
+                    cc1352p-launchpad \
+                    cc2650-launchpad \
+                    cc2650stk \
+                    fox \
+                    hifive1 \
+                    hifive1b \
+                    iotlab-a8-m3 \
+                    iotlab-m3 \
+                    limifrog-v1 \
+                    lobaro-lorabox \
+                    maple-mini \
+                    mbed_lpc1768 \
+                    msb-430 \
+                    msb-430h \
+                    nucleo-f103rb \
+                    nucleo-f446ze \
+                    nucleo-g070rb \
+                    nucleo-g071rb \
+                    nucleo-wl55jc \
+                    opencm904 \
+                    seeeduino_arch-pro \
+                    spark-core \
+                    stm32f3discovery \
+                    stm32mp157c-dk2 \
+                    teensy31 \
+                    telosb \
+                    z1 \
+                    #
```

Comment:
Why is this needed? Wouldn't the boards with missing `periph_adc` and `periph_hwrng` just be ignored by not meeting the `FEATURES_REQUIRED`?

Reply:
These are no hard feature requirements. This is why we made them `FEATURES_OPTIONAL` in the Makefile.dep. The boards listed have neither and ADC (exposed) nor a HWRNG

Reply:
Could you check if the following works?
```
FEATURES_REQUIRED_ANY += periph_adc|periph_hwrng
FEATURES_OPTIONAL += periph_adc periph_hwrng
```

The first line checks for at least one of them, but prioritizes the first one (i.e. when both are present `periph_hwrng` would not be used). The second line tries to enable both if provided. When none is provided this would fail.